src/VideoNativeComponent.ts

```diff
@@ -134,86 +139,93 @@ export type OnSeekData = Readonly<{
   currentTime: number;
   seekTime: number;
   finished: boolean;
-}>
+}>;
 
 export type OnPlaybackStateChangedData = Readonly<{
   isPlaying: boolean;
-}>
+}>;
 
 export type OnTimedMetadataData = Readonly<{
-  metadata: ReadonlyArray<Readonly<{
-    value?: string
-    identifier: string
-  }>>
-}>
-
+  metadata: ReadonlyArray<
+    Readonly<{
+      value?: string;
+      identifier: string;
+    }>
+  >;
+}>;
 
 export type OnAudioTracksData = Readonly<{
-  audioTracks: ReadonlyArray<Readonly<{
-    index: number
-    title?: string
-    language?: string
-    bitrate?: number
-    type?: string
-    selected?: boolean
-  }>>
-}>
+  audioTracks: ReadonlyArray<
+    Readonly<{
+      index: number;
+      title?: string;
+      language?: string;
+      bitrate?: number;
+      type?: string;
+      selected?: boolean;
+    }>
+  >;
+}>;
 
 export type OnTextTracksData = Readonly<{
-  textTracks: ReadonlyArray<Readonly<{
-    index: number
-    title?: string
-    language?: string
-    /**
-     * iOS only supports VTT, Android supports all 3
-     */
-    type?: 'srt' | 'ttml' | 'vtt'
-    selected?: boolean
-  }>>
-}>
+  textTracks: ReadonlyArray<
+    Readonly<{
+      index: number;
+      title?: string;
+      language?: string;
+      /**
+       * iOS only supports VTT, Android supports all 3
+       */
+      type?: 'srt' | 'ttml' | 'vtt';
+      selected?: boolean;
+    }>
+  >;
+}>;
 
 export type OnVideoTracksData = Readonly<{
-  videoTracks: ReadonlyArray<Readonly<{
-    trackId: number
-    codecs?: string
-    width?: number
-    height?: number
-    bitrate?: number
-    selected?: boolean
-  }>>
-}>
+  videoTracks: ReadonlyArray<
+    Readonly<{
+      trackId: number;
+      codecs?: string;
+      width?: number;
+      height?: number;
+      bitrate?: number;
+      selected?: boolean;
+    }>
+  >;
+}>;
 
 export type OnPlaybackData = Readonly<{
   playbackRate: number;
 }>;
 
 export type OnExternalPlaybackChangeData = Readonly<{
   isExternalPlaybackActive: boolean;
-}>
+}>;
 
 export type OnGetLicenseData = Readonly<{
   licenseUrl: string;
   contentId: string;
   spcBase64: string;
-}>
+}>;
 
 export type OnPictureInPictureStatusChangedData = Readonly<{
   isActive: boolean;
-}>
+}>;
 
 export type OnReceiveAdEventData = Readonly<{
   event: string;
-}>
+}>;
 
 export type OnVideoErrorData = Readonly<{
   error: string;
-}>
+}>;
 
 export type OnAudioFocusChangedData = Readonly<{
   hasFocus: boolean;
-}>
+}>;
 
-export type NativeVideoResizeMode = 'ScaleNone' | 'ScaleToFill' | 'ScaleAspectFit' | 'ScaleAspectFill';
```

Comment:
It because in native it is not like that - in android it is int and on iOS it is else something else. I guess i will refactor this to dont export constants, but create string type and handle it to return responding value on native side